### ISSUE TITLE
Shuffle input data during training

### ISSUE THREAD

**Issue by RAGoody** [RAGoody/neuralNetModeler#4]
As a user training on a data set, I want to make sure the training data is shuffled after each epoch.

UAC
- the data processed for each epoch is in a different row order than the proceeding epochs.
